Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TabView/TabViewXaml.bind

```diff
@@ -0,0 +1,102 @@
+<Page xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:ex="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+      mc:Ignorable="d">
+  
+  <Page.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="ms-appx:///Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.xaml" />
+      </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+  </Page.Resources>
+
+  <Grid>
+    <Grid.RowDefinitions>
+      <RowDefinition/>
+      <RowDefinition Height="Auto"/>
+    </Grid.RowDefinitions>
+    <controls:TabView x:Name="Tabs">
```

Comment:
Why two tabs in one sample? I recommend you simplify to one tab. Bind properties to the bind file to change behavior

Reply:
Yeah, I meant TabViews, thanks.

I agree this scenario is beneficial , but this sounds like a great second sample (once we support multisamples). IMO, it's more important to show a simpler demo that is easier to get started instead of trying to show everything that is possible.

Reply:
Yeah, I agree, it's just been my testing bed, so I need to simplify it a bit after all the outstanding issues are resolved.  I may expose some of the flags in the sample's properties panel, e.g. for enabling closing and dragging, etc...